### PR TITLE
Doc/omi roo 04/update readme

### PR DESCRIPTION
**Pull request by yoniihdc**
#### What does this PR do?
- A better description of the project.

#### Where should the reviewer start?
- Review the README.md file
:checkered_flag:

#### How should this be manually tested?
1. Reading the README.md file

#### Any background context you want to provide?
This PR depends on #14

### CODE REVIEW

**Comment by yoniihdc** [comment]
@Mart-97 the description looks good to me, the only problem is the style of your PR, for each section, you can replace or erase the default information.

ie, 
```markdown
## What does this PR do?
This PR provides a better description of the project.

...
## Any background context you want to provide
This PR depends on #14

## What are relevant tickets
[Issue 4](https://github.com/cancunDevs/omiqroo-front/issues/4)
```
If there is any section that you don't need just remove it from the PR template.

**Comment by yoniihdc** [comment]
@PabloFer13  other files are related to #14  that still WIP.

**Comment by PabloFer13** [comment]
OK.

File: README.md [hunk 1]
```diff
@@ -1,28 +1,12 @@
...
-
-# Components
-* [Babel](https://babeljs.io/)
-* [ESlint](https://eslint.org/)
-* [React](https://reactjs.org/)
-* [Webpack](https://webpack.js.org/)
-* [SASS](http://sass-lang.com/)
+# OMI-QROO-FRONT
+
+OMI-QROO-FRONT is a project that aims,to report all the relevant information about the Quintana Roo Olympiad in Informatics, this project follows the Style Guide of [Airbnb's Javascript (ES6)] (https://github.com/airbnb/javascript), and includes some features that allow it use SASS as your style tool in the application.
```
Comment (yoniihdc) [inline README.md]: You need to remove blank space between `[Airbnb's Javascript (ES6)](https://github.com/airbnb/javascript)` on `] (`

Comment (yoniihdc) [reply README.md]: @Mart-97 ^

**Comment by yoniihdc** [comment]
LGTM 💯

**Pull request merged**